x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities.tsx

```diff
@@ -4,17 +4,236 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { EuiCallOut, EuiSpacer } from '@elastic/eui';
-import React from 'react';
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiDataGrid,
+  EuiEmptyPrompt,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  useEuiTheme,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { DataView } from '@kbn/data-views-plugin/common';
+import React, { useMemo } from 'react';
+import { LOCAL_STORAGE_PAGE_SIZE_FINDINGS_KEY } from '../../common/constants';
+import { useCloudPostureTable } from '../../common/hooks/use_cloud_posture_table';
+import { useLatestVulnerabilities } from './hooks/use_latest_vulnerabilities';
+import { VulnerabilityRecord } from './types';
+import { getVulnerabilitiesColumns } from './utils';
+import { LATEST_VULNERABILITIES_INDEX_PATTERN } from '../../../common/constants';
+import { ErrorCallout } from '../configurations/layout/error_callout';
+import { FindingsSearchBar } from '../configurations/layout/findings_search_bar';
+import { useFilteredDataView } from '../../common/api/use_filtered_data_view';
+import { CVSScoreBadge, SeverityStatusBadge } from '../../components/vulnerability_badges';
+
+const getDefaultQuery = ({ query, filters }: any): any => ({
+  query,
+  filters,
+  sort: [{ id: 'cvss', direction: 'desc' }],
```

Comment:
Same as the comments above. Storing `id` as constant unless `sort.id` has specified types